src/app/components/our-team/our-team.component.scss

```diff
@@ -140,18 +141,34 @@
       text-align: right;
       padding-left: 50px;
       right: 15px;
+      .text-dark-black {
+          color: $dark-black;
+          font-size: $fs-15;
+          font-weight: $fw-semibold;
+      }
+      .text-med-black {
+          color: $dark-black;
+          font-size: $fs-13;
+      }
       h4 {
-          font-size: 16px;
+          font-size: $fs-16;
       }
   }
 
   .team-link-head {
-      padding: 10px 15px;
+      padding: 10px 5px 10px 5px !important;
```

Comment:
I see at many places you using `!important` (in other PR's as well). Generally, it's discouraged to use this and use only if we just can't do without this. Was this the case here?

Reply:
@lunayach no, previously it was `padding-left` from the some other place, so just to override I used it but it is not required as I've removed that `padding-left`

Removed it now, please review